fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/catalog/FlinkCatalogITCase.java

```diff
@@ -279,6 +286,124 @@ void testAutoPartitionedTable() throws Exception {
         assertResultsIgnoreOrder(showPartitionIterator, expectedShowPartitionsResult, true);
     }
 
+    @Test
+    void testAutoPartitionedTableWithMultiPartitionKeys() throws Exception {
+        ObjectPath objectPath =
+                new ObjectPath(DEFAULT_DB, "test_auto_partitioned_table_with_multi_partition_keys");
+
+        // 1. test invalid auto partition table.
+        // not specify auto partition key
+        assertThatThrownBy(
+                        () ->
+                                tEnv.executeSql(
+                                        "create table test_auto_partitioned_table_with_multi_partition_keys (a int, b string, c string, dt string) partitioned by (b,c,dt) "
+                                                + "with ('table.auto-partition.enabled' = 'true',"
+                                                + " 'table.auto-partition.time-unit' = 'day',"
+                                                + " 'table.auto-partition.num-precreate' = '0'"
+                                                + ")"))
+                .cause()
+                .isInstanceOf(CatalogException.class)
+                .hasMessage(
+                        "Failed to create table fluss.test_auto_partitioned_table_with_multi_partition_keys in testcatalog");
+
+        // specified auto partition key not in partition keys
+        assertThatThrownBy(
+                        () ->
+                                tEnv.executeSql(
+                                        "create table test_auto_partitioned_table_with_multi_partition_keys (a int, b string, c string, dt string) partitioned by (b,c,dt) "
+                                                + "with ('table.auto-partition.enabled' = 'true',"
+                                                + " 'table.auto-partition.time-unit' = 'day',"
+                                                + " 'table.auto-partition.key' = 'a',"
+                                                + " 'table.auto-partition.num-precreate' = '0'"
+                                                + ")"))
+                .cause()
+                .isInstanceOf(CatalogException.class)
+                .hasMessage(
+                        "Failed to create table fluss.test_auto_partitioned_table_with_multi_partition_keys in testcatalog");
```

Comment:
Please assert the key exception message, e.g., `The specified key for auto partitioned table is not a partition key`.

Currently, all the 3 expected exception message are equal which may hide problems.